packages/language-server/src/prisma-fmt/referentialActions.ts

```diff
@@ -0,0 +1,23 @@
+import execa from 'execa'
+
+export default function referentialActions(
+  execPath: string,
+  text: string,
+  onError?: (errorMessage: string) => void,
+): string[] {
+  const result = execa.sync(execPath, ['referential-actions'], { input: text })
+
+  if (result.exitCode !== 0) {
+    const errorMessage =
+      "prisma-fmt error'd during getting available preview features.\n"
```

Comment:
Hmm Is this the right error message? I may be misunderstanding the function

Reply:
No, that is probably copy/paste :D